### PR TITLE
Skip merge ID set step in contrib

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/32703

## Description
Skip merging ID set step in contrib.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -316,17 +316,25 @@ references:
...
 
         # Download private ID set
         gsutil cp gs://marketplace-dist/content/private_id_set.json $CIRCLE_ARTIFACTS/unified_id_set.json
+        echo "successfully download private ID set"
         gcloud auth revoke --all
         rm $GCS_PATH
 
         # Merge public and private ID sets
         demisto-sdk merge-id-sets -i1 ./Tests/id_set.json -i2 $CIRCLE_ARTIFACTS/unified_id_set.json -o $CIRCLE_ARTIFACTS/unified_id_set.json
+        echo "successfully merge public and private ID sets"
```
Comment (hod-alpert) [inline .circleci/config.yml:337]:
```suggestion
        echo "successfully merged public and private ID sets"
```

**Review by hod-alpert — APPROVED** [review]
Just a small comment

File: .circleci/config.yml (hunk 1 above)
Comment (ChanochShayner) [reply .circleci/config.yml]: done

File: .circleci/config.yml [hunk 2]
```diff
@@ -316,17 +316,25 @@ references:
...
       command: |
+        if [[ $CIRCLE_BRANCH =~ pull/[0-9]+ ]]; then
+            echo "Skipping, Should not run on contributor's branch."
+            exit 0
+        fi
+
         GCS_PATH=$(mktemp)
         echo "$GCS_MARKET_KEY" > "$GCS_PATH"
         gcloud auth activate-service-account --key-file="$GCS_PATH" > auth.out 2>&1
+        echo "successfully activate service account"
```
Comment (guykeller) [inline .circleci/config.yml:327]:
```suggestion
        echo "successfully activated google cloud service account"
```

File: .circleci/config.yml [hunk 3]
```diff
@@ -316,17 +316,25 @@ references:
...
 
         # Download private ID set
         gsutil cp gs://marketplace-dist/content/private_id_set.json $CIRCLE_ARTIFACTS/unified_id_set.json
+        echo "successfully download private ID set"
```
Comment (guykeller) [inline .circleci/config.yml:331]:
```suggestion
        echo "successfully downloaded private ID set"
```

**Review by guykeller — APPROVED** [review]
few minor suggesstions

**Pull request merged**